hyacinth/session.py

```diff
@@ -554,3 +555,14 @@ def delete_custom_action(self, id, **kwargs):
         """DELETE an existing Custom Action with provided ID."""
         url = Session.__make_url(f"custom_actions/{id}")
         return self.__delete_resource(url, **kwargs)
+
+    def verify_custom_action(self, subject_url, custom_action_nonce, **kwargs):
+        """Verify a Custom Action."""
+        url = CLIO_BASE_URL_US + subject_url
+        params = kwargs.get("params")
```

Comment:
```suggestion
        params = kwargs.get("params", {})
```
will fix a `KeyError` if caller doesn't pass `params`